chromeapp/test/scripts/dnssd/dns-util-test.js

```diff
@@ -0,0 +1,220 @@
+'use strict';
```

Comment:
Consider adding tests to cover the various error cases - you seem to mostly be testing the sane cases here. Also suggest testing for domains with non-ASCII Unicode characters. You can do this in a follow-up CL if you like.